op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -69,6 +69,10 @@ var taggedReleases = map[string]TaggedRelease{
 		ArtifactsHash: common.HexToHash("78f186df4e9a02a6421bd9c3641b281e297535140967faa428c938286923976a"),
 		ContentHash:   common.HexToHash("361ebf1f520c20d932695b00babfff6923ce2530cd05b2776eb74e07038898a6"),
 	},
+	"op-contracts/v2.0.0-rc.1": {
```

Comment:
Why not use a string `const` here as we do for the others?

Reply:
The consts refer to the "latest" version in the lineage. This is here so that users can continue using the RC if they need to.